src/_nested-list.scss

```diff
@@ -81,6 +85,18 @@ $button: #{$fd-namespace}-button;
         border-top: none;
       }
     }
+
+    &--popover {
+      &:first-child {
+        border-top-left-radius: 0.25rem;
```

Comment:
can we keep this value in a variable and reuse it?

Reply:
using variable now.